### PR TITLE
Enable to build release package on i386/arm

### PR DESCRIPTION
**Pull request by k-okada**
- CMAKE_SYSTEM_PROCESSOR could not returns correct host architecture on Docker container, so changed to use CMAKE_LIBRARY_ARCHITECTURE 
  - https://stackoverflow.com/questions/31012297/uname-a-returning-the-same-in-docker-host-or-any-docker-container
  - Note that CMAKE_SYSTEM_PROCESSOR uses `uname -p` https://cmake.org/cmake/help/v3.1/variable/CMAKE_SYSTEM_PROCESSOR.html and. https://cmake.org/cmake/help/v3.1/variable/CMAKE_HOST_SYSTEM_PROCESSOR.html#variable:CMAKE_HOST_SYSTEM_PROCESSOR
- To test on i386/arm architecture on travis,  change `ARCH` value in .travis file

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -36,6 +41,8 @@ install:
   - cp -R $TRAVIS_BUILD_DIR $JOB_PATH/catkin_workspace/src/
   # generate the script to run a pre-release job for that target and repo
   - generate_prerelease_script.py https://raw.githubusercontent.com/ros-infrastructure/ros_buildfarm_config/production/index.yaml $ROS_DISTRO_NAME default $OS_NAME $OS_CODE_NAME $ARCH  --output-dir $JOB_PATH
+  # skip build and test for ARM containers
+  - if [[ "$ARCH" == "arm"* ]]; then sed -i '/build and test/i mkdir -p $WORKSPACE/catkin_workspace/test_results; exit 0' $JOB_PATH/prerelease_build_underlay.sh; fi
```
Comment (d-nakamichi) [inline .travis.yml]: What is the reason adding this?
Is it for timeout of travis?

**Comment by k-okada** [comment]
yes, running arm vm is slow compared to the native architecture and I think
we should add "--skip-test" option to 'generate_prerelease_script ' in
future.

--
◉ Kei Okada



2019年5月29日(水) 10:41 Daisuke NAKAMICHI <notifications@github.com>:

File: .travis.yml (hunk 1 above)
Comment (d-nakamichi) [reply .travis.yml]: OK, I agree.

**Comment by d-nakamichi** [comment]
Thank you! @k-okada

**Pull request merged**